onnxruntime/python/tools/transformers/benchmark.py

```diff
@@ -262,7 +262,12 @@ def run_onnxruntime(
                         "datetime": str(datetime.now()),
                     }
 
-                    logger.info(f"Run onnxruntime on {model_name} with input shape {[batch_size, sequence_length]}")
+                    if config.model_type == "vit" or config.model_type == "swin":
```

Comment:
change `config.model_type == "vit" or config.model_type == "swin"` to `config.model_type in ["vit", "swin"]`, and in other places as well.